src/dis.js

```diff
@@ -0,0 +1,57 @@
+import {
+    getTodoList, removeItem, editItem, markCompleted,
+  } from './todolist.js';
+  
+  const listContainer = document.querySelector('.list');
+  const displayToDoList = (todoList) => {
+    listContainer.innerHTML = '';
+    todoList.forEach((item, index) => {
+      const listItem = document.createElement('div');
+      listItem.className = 'properties-list';
+      listItem.innerHTML = `
+            <div class="sides">
+              <span class="items-left">
+                <input type="checkbox" ${item.completed ? 'checked' : ''} data-index="${index}">
+                <input type="text" value="${item.description}" class="input2">
+              </span>
+              <span class="items-right">
+                <div class="edit-item" data-index="${index}"></div>
+                <div class="remove-item" data-index="${index}"><i class="fa-solid fa-trash"></i></div>
+              </span>
+            </div>
+            <hr class="line-one">
+          `;
```

Comment:
Please use an alternative to innerHTML, such as creating the elements using createElement and then appending them to the listItem element.This method is more verbose but can be easier to read and maintain than using innerHTML. It also avoids some potential security issues that can arise when using innerHTML.(Java Script best practice)